sonar-php-plugin/src/test/java/org/sonar/plugins/php/PhpTestUtils.java

```diff
@@ -32,7 +32,7 @@
 public class PhpTestUtils {
 
   public static final String PHPUNIT_REPORT_DIR = "org/sonar/plugins/php/phpunit/sensor/";
```

Comment:
@inverno I think path should be changed (it's not sensor anymore) and simplified may be?:)